### ISSUE TITLE
TypeError: Cannot set property 'readableListening' of undefined

### ISSUE THREAD

**Issue by talius-gti** [adashen/vscode-tomcat#280]
- VSCode Version: 1.43
- OS Version: Ubuntu LTS 18.04
- Tomcat Extension Version: 0.11.1

Steps to Reproduce:

1. Run a war from a folder or from the Tomcat container.

I never had this problem in several months of usage, until few days ago. It popped out from nowhere.

**Comment by hsdonim82** [comment]
I have the same problem. 
run a war file in tomcat -> TypeError: Cannot set property 'readableListening' of undefined

VSCode Version: 1.43.1
OS Version: Ubuntu LTS 18.04
Tomcat Extension Version: 0.11.1

please tell me how to solve this problem~!

**Comment by rriopel432** [comment]
Same problem.  Worked earlier last week when I was setting up a new Mac using Remote Containers.  Went to use it the next day and now I'm getting that error.  Any tips on troubleshooting?

I tried recreating the container and now I can't add the Tomcat server.

**Comment by talius-gti** [comment]
I switched plugin in the end 😬 now I use Just Start Server, for anyone searching for a solution.

**Comment by DeathPoem** [comment]
I'm using remote-vscode, and got same error when type start server on 'tomact server'. Restart remote linux and rm ./.vscode-server/ -rf then reinstall extension fix problem.

**Comment by WayenVan** [comment]
is this a problem only occur when using remote-vscode?  reinstall extension may fix the problem

**Comment by andxu** [comment]
@talius-gti We have not released this extension for years (due to lack resource), so it apparently is caused by dependency change. Is it reoccur on remote container stably?

**Comment by obarbier** [comment]
> on remote container stably?

yes this is a reoccurring solution. I can take a look at it with some guidance

**Comment by obarbier** [comment]
> @talius-gti We have not released this extension for years (due to lack resource), so it apparently is caused by dependency change. Is it reoccur on remote container stably?

yes this is a reoccurring solution. I can take a look at it with some guidance

**Comment by hogan63** [comment]
sad, it still has this problem, yesterday i run i all ok, but today it shows this problem same as you , i run my vscode on remote linux server  , where my tomcat for java version is **v0.11.3** , my VsCode info:
Version: 1.50.1
Commit: d2e414d9e4239a252d1ab117bd7067f125afd80a
Date: 2020-10-13T14:53:05.704Z
Electron: 9.2.1
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0

**Comment by hogan63** [comment]
> sad, it still has this problem, yesterday i run i all ok, but today it shows this problem same as you , i run my vscode on remote linux server , where my tomcat for java version is **v0.11.3** , my VsCode info:
> Version: 1.50.1
> Commit: d2e414d9e4239a252d1ab117bd7067f125afd80a
> Date: 2020-10-13T14:53:05.704Z
> Electron: 9.2.1
> Chrome: 83.0.4103.122
> Node.js: 12.14.1
> V8: 8.3.110.13-electron.0
> OS: Darwin x64 19.6.0

i then reintalled another version 0.11.1, then it works fine.

**Comment by flwzz** [comment]
the same problem. I finally give up to run my project on vscode!

**Comment by raul-leon** [comment]
I'm sorry for my English.

Searching for Tomcat in VSCode settings and disable **Tomcat: Restart_when_http(s)_port_change** works for me.

**Comment by ozyx** [comment]
Quick fix for me has been to do a "Reload Window" from the VSCode command menu. Inconvenient but at least gets it working again.

**Comment by ozyx** [comment]
> Quick fix for me has been to do a "Reload Window" from the VSCode command menu. Inconvenient but at least gets it working again.

This solution does not work on MacOS Big Sur

**Comment by RiccardoBarbieri** [comment]
Has anyone found a fix that is not using another extension or reinstalling it?
